lib/main.js

```diff
@@ -157,6 +158,18 @@ app.use(function(req, res, next) {
 // Set up 
 var assets = connect.static(paths.projectFile('assets'),
                             {maxAge:86400000}); // one day
+// Precompile JS template
+var ssjsTemplate = fs.readFileSync(paths.projectFile('assets/shiny-server.js'), 'utf-8');
```

Comment:
I'm OK with this. I asked because it's different from our asset/template separation elsewhere.